### PR TITLE
Add Wayland components and replace build pipeline to Meson

### LINKED ISSUE #39: Bump build number so we don't forget
### Additional comment:

_No response_

### PR DESCRIPTION
**Pull request by ehfd**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

Closes #40
 
Closes #39

### CODE REVIEW

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

**Comment by ehfd** [comment]
@hmaarrfk It seems that Wayland libraries can actually built since `wayland-scanner` is inside the Wayland feedstock.

Now... what should be done here so that the `libxcb` mess doesn't recur?

**Comment by ehfd** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -45,11 +46,23 @@ test:
     - test -f $PREFIX/lib/libva.so
     - test -f $PREFIX/lib/libva.so.2
     - test -f $PREFIX/lib/pkgconfig/libva.pc
-    - test -f $PREFIX/lib/pkgconfig/libva-drm.pc
```
Comment (hmaarrfk) [inline recipe/meta.yaml:48]: why is this file no longer needed, the `libva-drm.pc`

Comment (ehfd) [reply recipe/meta.yaml:48]: I moved it to below.

**Comment by ehfd** [comment]
I'm hoping `libxcb` doesn't break.

**Comment by ehfd** [comment]
`libxcb:          1.16-h7935292_0      conda-forge`

@hmaarrfk Not good at all. Need resolution promptly.

**Pull request merged**